Packages/com.unity.inputsystem/InputSystem/InputManager.cs

```diff
@@ -88,6 +88,14 @@ public InputUpdateType updateMask
             }
         }
 
+        #if UNITY_EDITOR
+        public bool oldInputWarnUserFlag
```

Comment:
This is better kept directly on InputSystemObject and should have a better name.